pages.da/common/sed.md

```diff
@@ -0,0 +1,32 @@
+# sed
+
+> Rediger tekst, programmatisk.
+> Mere information: <https://www.gnu.org/software/sed/manual/sed.html>.
+
+- Erstat den første forekomst af et regulært udtryk (regular expression) i hver linje af en fil, og print resultatet:
+
+`sed 's/{{regular_expression}}/{{erstat}}/' {{filnavn}}`
+
+- Erstat alle forekomster af et regulært udtryk in en fil, og print resultatet:
+
+`sed -r 's/{{regular_expression}}/{{erstat}}/g' {{filnavn}}`
+
+- Erstat alle forekomster af en streng i en fil, og overskriv filen:
+
+`sed -i 's/{{find}}/{{erstat}}/g' {{filnavn}}`
+
+- Erstat forekomster på linjer der matcher mønstret:
+
+`sed '/{{linje_mønster}}/s/{{find}}/{{erstat}}/' {{filnavn}}`
+
+- Fjern linjer der matcher mønstret:
+
+`sed '/{{linje_mønster}}/d' {{filnavn}}`
+
+- Print de første 11 linjer af en fil:
+
+`sed 11q {{filnavn}}`
+
+- Udfør flere find-og-erstat udtryk i en fil:
+
+`sed -e 's/{{find}}/{{erstat}}/' -e 's/{{find}}/{{erstat}}/' {{filnavn}}`
```

Comment:
This command is missing 
```bash
  - Replace separator `/` by any other character not used in the find or replace patterns, e.g. `#`:
    sed 's#find#replace#' filename
```

Ref.
https://github.com/tldr-pages/tldr/blob/a69bc58fb80ff250245cad72e5e730dfe70aaba4/pages/common/sed.md?plain=1#L36

Reply:
Added the command, don't know how I missed it. 😬